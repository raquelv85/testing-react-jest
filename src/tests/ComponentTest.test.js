


describe('Pruebas en <ComponentTest/>',() => {
  test('debe de mostrar el mensaje "Hola soy Raquel"',() => {

  })
})