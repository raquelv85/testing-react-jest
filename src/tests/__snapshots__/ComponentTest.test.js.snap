// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <ComponentTest/> debe de mostrar <ComponentTest/> correctamente 1`] = `
<Fragment>
  <h1>
    Hola soy Raquel
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
